pkgs/development/python-modules/connection-pool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "connection-pool";
+  version = "0.0.3";
+  disabled = !isPy3k;
+
+  # tests are missing from the PyPi tarball
```

Comment:
Why not fetch from Github then?

Reply:
This comment is the unwanted one. Seems like it just got copied over from stopit.